internal/resources/syntheticmonitoring/resource_check_alert.go

```diff
@@ -0,0 +1,226 @@
+package syntheticmonitoring
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	smapi "github.com/grafana/synthetic-monitoring-api-go-client"
+	"github.com/grafana/synthetic-monitoring-api-go-client/model"
+	"github.com/grafana/terraform-provider-grafana/v3/internal/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+var resourceCheckAlertID = common.NewResourceID(common.IntIDField("check_id"))
+
+const (
+	// Alert names
+	AlertNameProbeFailedExecutionsTooHigh        = "ProbeFailedExecutionsTooHigh"
+	AlertNameTLSTargetCertificateCloseToExpiring = "TLSTargetCertificateCloseToExpiring"
+)
+
+func resourceCheckAlerts() *common.Resource {
+	return common.NewLegacySDKResource(
+		common.CategorySyntheticMonitoring,
+		"grafana_synthetic_monitoring_check_alerts",
+		resourceCheckAlertID,
+		&schema.Resource{
+			Description: `
+Manages alerts for a check in Grafana Synthetic Monitoring.
+
+* [Official documentation](https://grafana.com/docs/grafana-cloud/synthetic-monitoring/configure-alerts/)
```

Comment:
I think this link is incorrect:
```suggestion
* [Official documentation](https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/configure-alerts/configure-per-check-alerts/)
```